service/controller/azure/v22/version_bundle.go

```diff
@@ -8,8 +8,18 @@ func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
 			{
-				Component:   "TODO",
-				Description: "Add your changes here.",
+				Component:   "cert-exporter",
+				Description: "Migrated to use default app catalog.",
+				Kind:        versionbundle.KindChanged,
+			},
+			{
+				Component:   "net-exporter",
+				Description: "Migrated to use default app catalog.",
+				Kind:        versionbundle.KindChanged,
+		ted to v0.4.1. https://github.com/giantswarm/metrics-server-app/blob/db61c5c13c4ba55c357ab098253b3f3fe8f4e2cd/CHANGELOG.md#v041",
```

Comment:
This line needs to be removed.

Reply:
Badly resolved merge conflict, fixed it, PTAL